server/mailet/mailetcontainer-camel/src/test/java/org/apache/james/mailetcontainer/lib/AbstractStateMailetProcessorTest.java

```diff
@@ -163,7 +164,7 @@ public void afterMailet(Mailet m, String mailName, String state, long processTim
     }
 
     @Test
-    public void testMatcherThrowException() throws Exception {
+    public void testMatcherNoThrowException() throws Exception {
```

Comment:
We generally tend to name the tests we (re) write like this:

```
methodShouldAction[WhenCondition]
```

Here very likely:

```
matcherProcessingShouldNotResultInAnExceptionWhenMatcherThrows
```

Reply:
Okay, I will change that, and do the same for testMailetNoThrowException.